### ISSUE TITLE
LDR purpose and value/type

### ISSUE THREAD

**Issue by nagyrobi** [Strooom/SmartTherm#10]
What's the purpose of `Ext LDR` in the schematic (light sensor maybe) and what is the exact type of component to be used?

**Comment by Strooom** [comment]
You can connect a light sensor, so the SW could detect day / night from this.
Sensor is something like this : https://www.gotron.be/ldr-photoresistor-100mw-50-160kohm.html
I have never implemented this so I do not have an exact type but it's basically a voltage divider, reading it it via the ADC to determine how much light there is.

**Comment by nagyrobi** [comment]
Ok, so a photoresistor of 50-160 kOhm is what was intended to be used?

**Comment by jensweimann** [comment]
Yes, this will do. I have a similiar one working on the esphome config.
